### PR TITLE
feat: add new `node` command to `fuels` CLI

### LINKED ISSUE #2355: Adding new `node` command to `fuels` CLI
One may want to start a node in isolation, using _**the same configs**_ used by the `fuels dev`.

It should use the default configs or the user's configs if they specify them in `fuels.config.ts.`

### PR DESCRIPTION
**Pull request by Dhaiwat10**
Closes #2355

### CODE REVIEW

File: packages/fuels/src/cli.ts [hunk 1]
```diff
@@ -66,6 +67,11 @@ export const configureCli = () => {
     .addOption(pathOption)
     .action(withConfig(command, Commands.dev, dev));
 
+  (command = program.command(Commands.node))
+    .description('Start a Fuel node')
```
Comment (danielbate) [inline packages/fuels/src/cli.ts:71]:
```suggestion
    .description('Start a Fuel-Core node')
```

**Review by danielbate — APPROVED** [review]
🔥

File: packages/fuels/src/cli.ts (hunk 1 above)
Comment (nedsalk) [reply packages/fuels/src/cli.ts:71]: I suppose it would be `fuel-core` then.

Comment (Dhaiwat10) [reply packages/fuels/src/cli.ts:71]: The description for the `dev` command says `Fuel`, so I went with that here as well.

File: packages/fuels/src/cli/commands/node/index.ts
[the remark this answers is not in the document]
Comment (Dhaiwat10) [reply packages/fuels/src/cli/commands/node/index.ts:1]: Do you think it's worth it? It only watches for any changes in the config file. I think that's something everyone would want, and is a good default convention that is used by projects like Next.js for their config files as well.

**Comment by Dhaiwat10** [comment]
@petertonysmith94 added to readme: https://github.com/FuelLabs/fuels-ts/pull/2376/commits/643f9c2e1a3942abdf238eeca6d0d171d8ac1b72

File: packages/fuels/src/cli.ts (hunk 1 above)
Comment (maschad) [reply packages/fuels/src/cli.ts:71]: I think `Fuel` node would be more intelligible by the users of this command @danielbate , whilst we do make that distinction within the naming of the packages, there is only one implication of a _Fuel node_ to the best of my knowledge.

**Review by maschad — APPROVED** [review]
Great job on this!

**Review by danielbate — APPROVED** [review]

**Pull request merged**